### PR TITLE
fix(sniplet): #RBS-129 fix sniplet surplus calls

### PR DESCRIPTION
**Pull request by alicedraillard**
## Describe your changes
the two instances of the sniplet are initialized at the same time as the module but when one is displayed information is sent to the opened sniplet

## Checklist tests
check ng2 console and network when using sniplet (changing date/resource type ...)

## Issue ticket number and link
#RBS-129 https://entsupport.gdapublic.fr/browse/RBS-129

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts [hunk 1]
```diff
@@ -171,21 +174,21 @@ class ViewModel implements IViewModel {
      * Get calendarEvent change and change form accordingly
      */
     setHandler(): void {
-        this.scope.$on(RBS_CALENDAR_EVENTER.INIT_BOOKING_INFOS, (event: IAngularEvent, calendarEvent: CalendarEvent) => {
+        this.scope.$on(this.source + RBS_CALENDAR_EVENTER.INIT_BOOKING_INFOS, (event: IAngularEvent, calendarEvent: CalendarEvent) => {
+            console.log("init", this.source + RBS_CALENDAR_EVENTER.INIT_BOOKING_INFOS);
```
Comment (alicedraillard) [inline src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts]: will be deleted, for PO tests

File: src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts [hunk 2]
```diff
@@ -171,21 +174,21 @@ class ViewModel implements IViewModel {
...
+            this.canEditEvent = (this.source == editionSniplet);
             this.handleCalendarEventChange(event, calendarEvent);
         });
 
-        this.scope.$on(RBS_CALENDAR_EVENTER.UPDATE_BOOKING_INFOS, (event: IAngularEvent, calendarEvent: CalendarEvent) => {
+        this.scope.$on(this.source + RBS_CALENDAR_EVENTER.UPDATE_BOOKING_INFOS, (event: IAngularEvent, calendarEvent: CalendarEvent) => {
+            console.log("update", this.source + RBS_CALENDAR_EVENTER.UPDATE_BOOKING_INFOS);
```
Comment (alicedraillard) [inline src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts]: will be deleted, for PO tests

File: src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts [hunk 3]
```diff
@@ -171,21 +174,21 @@ class ViewModel implements IViewModel {
...
             this.handleCalendarEventChange(event, calendarEvent);
         });
 
-        this.scope.$on(RBS_CALENDAR_EVENTER.CLOSE_BOOKING_INFOS, (event: IAngularEvent) => {
+        this.scope.$on(this.source + RBS_CALENDAR_EVENTER.CLOSE_BOOKING_INFOS, (event: IAngularEvent) => {
+            console.log("close", this.source + RBS_CALENDAR_EVENTER.CLOSE_BOOKING_INFOS);
```
Comment (alicedraillard) [inline src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts]: will be deleted, for PO tests

File: src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts [hunk 4]
```diff
@@ -397,7 +400,7 @@ class ViewModel implements IViewModel {
         let checkedStartMoment = moment(moment(this.calendarEvent.startMoment)
             .format(this.dateFormat.formattedDateTimeNoSeconds))
             .hours(this.calendarEvent.allday ? ALL_DAY.startHour :this.calendarEvent.startTime.getHours())
-            .minutes(this.calendarEvent.allday ? ALL_DAY.startHour :this.calendarEvent.startTime.getMinutes());
+            .minutes(this.calendarEvent.allday ? ALL_DAY.minutes :this.calendarEvent.startTime.getMinutes());
```
Comment (alicedraillard) [inline src/main/resources/public/ts/sniplets/calendar-rbs-booking.sniplet.ts:401]: quick fix

File: build.sh [hunk 5]
```diff
@@ -24,37 +24,36 @@ clean () {
 
 buildNode () {
   #jenkins
-  echo "[buildNode] Get branch name from jenkins env..."
```
Comment (alicedraillard) [inline build.sh]: For ng2, commit will be deleted
+all commits will be squashed at the end

**Review by BastienBenard — APPROVED** [review]

**Review by JinDouang — APPROVED** [review]

**Pull request merged**